ambry-api/src/main/java/com/github/ambry/quota/QuotaChargeCallback.java

```diff
@@ -68,6 +68,16 @@ public void chargeQuota(long chunkSize) throws RouterException {
       public void chargeQuota() throws RouterException {
         chargeQuota(quotaManager.getQuotaConfig().quotaAccountingUnit);
       }
+
+      @Override
+      public boolean shouldThrottle() {
```

Comment:
I would recommend to use the same method names as the Chargeable here, so we know the relation between the callback and the chargeable by just looking to the interface. We can probably rename the `chargeQuota` to `charge` and the `shouldThrottle` to `check`.

Reply:
done